Jenkinsfile

```diff
@@ -634,7 +634,7 @@ def withBeatsEnv(Map args = [:], Closure body) {
 
   if(isUnix()) {
     gox_flags = (isArm() && is64arm()) ? '-arch arm' : '-arch amd64'
-    path = "${env.WORKSPACE}/bin:${env.PATH}"
+    path = "${env.WORKSPACE}/bin:${env.PATH}:/usr/local/bin"
```

Comment:
At the moment the orka ephemeral workers don't have the `/usr/local/bin` in their PATH environment variable

```
[2021-12-15T10:23:58.372Z] PATH=/Users/admin/workspace/PR-29426-4-fae0730c-b250-44cb-907a-866a5b5b3d7b/bin:/usr/bin:/bin:/usr/sbin:/sbin
```

![image](https://user-images.githubusercontent.com/2871786/146210397-864bf04f-6608-4974-adc1-18856f1faadd.png)